travis/setup_conda.sh

```diff
@@ -48,11 +48,18 @@ fi
 if [ "${DOCS}" == true ];
  then export DOCSBUILD="sphinx sphinx_rtd_theme graphviz";
 fi
+
 echo "dependencies: ${MATPLOTLIB} ${NUMPY} ${FITS} ${XSPEC} ${DOCSBUILD}"
 
+# Hack to force AstroPy < 3.2 (see https://github.com/sherpa/sherpa/issues/632)
+# This is a short-term hack just to get travis tests to pass
+# Should there be better a way of specifying versions for these dependencies?
```

Comment:
This should be fine. A simple alternative might be to add the constraint to the yaml configuration, I think. In any case since this has to be fixed to support newer versions of astropy anyway, I think this fix is fine (unless Travis things otherwise for some unforeseen reason).

Reply:
The only place I could see to put it would be the `FITS=astropy` setting in `.travis.yml`, and I was concerned that even with `FITS='astropy<3.2'` it might not work (in particular since this then gets passed to the test code). Did you have something else in mind?

Not that I think it's worth it for this case, but it might be a useful feature to have (to be able to say "this is tested against AstroPy 3.1").